fetch.bs

```diff
@@ -4695,6 +4861,7 @@ dictionary RequestInit {
   RequestRedirect redirect;
   DOMString integrity;
   boolean keepalive;
+  AbortSignal? signal;
```

Comment:
Why is this nullable and the readonly attribute is not? What does null signify vs undefined?

Reply:
```js
const request = new Request(otherRequest);
```

If the signal passed to `otherRequest` aborts, so will `request`.

```js
const request = new Request(otherRequest, {signal: null});
```

If the signal passed to `otherRequest` aborts, `request` will not abort.

Reply:
And for the second scenario `request.signal` will still return an `AbortSignal`? That makes sense then, thanks.

Reply:
Yep!